test/run-on-many-lisps-and-openssls/openssl-releases/build.sh

```diff
@@ -10,7 +10,7 @@ cd ../src
 VERSIONS="$1"
 if [ -z "$VERSIONS" ]
 then
-  VERSIONS="openssl-0.9.8zh openssl-1.0.0s openssl-1.0.2q openssl-1.1.0j openssl-1.1.1a"
+  VERSIONS="openssl-0.9.8zh openssl-1.0.0s openssl-1.0.2q openssl-1.1.0j openssl-1.1.1a libressl-2.2.7 libressl-3.0.1"
 fi
```

Comment:
Why only two libressl versions by default?

Reply:
Simply failed to add them after having tested all versions thru Travis. Fixed.